### ISSUE TITLE
sbt-war.g8 template doesn't work as advertised

### ISSUE THREAD

**Issue by dylemma** [earldouglas/sbt-war#996]
I was planning to give the updated plugin a spin, so I generated a "hello world" app from the `earldouglas/sbt-war.g8` template. When I run `warStart` it shows 

```
sbt:hello-sbt-war> warStart
[info] [sbt-war] Starting server
[success] Total time: 1 s, completed Nov 15, 2024, 4:26:50 PM
```

But the server does not appear to actually be running:

```
curl: (7) Failed to connect to localhost port 8080 after 2231 ms: Could not connect to server
```

![image](https://github.com/user-attachments/assets/16a8c771-ba16-4bd0-93d7-ddbcc8731bc8)

```
sbt:hello-sbt-war> test
[error] Test mypackage.MyServletSuite.getRoot failed: java.lang.Exception: port8080 should be open, took 5.491 sec
[error]     at http.Request$.awaitOpen(http.scala:33)
[error]     at http.Request$.apply(http.scala:47)
[error]     at mypackage.MyServletSuite.getRoot(MyServletSuite.scala:22)
[error]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error]     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]     at java.lang.reflect.Method.invoke(Method.java:566)
[error]     ...
[error] Failed: Total 1, Failed 1, Errors 0, Passed 0
[error] Failed tests:
[error]         mypackage.MyServletSuite
[error] (Test / test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 6 s, completed Nov 15, 2024, 4:37:44 PM
```

OS: Windows 11
Java: 11

**Comment by earldouglas** [comment]
Thanks for the report!  I'll see if I can reproduce this in Windows.

**Comment by earldouglas** [comment]
Does making the forked JVM's output noisier give any interesting clues?

You can do this by either of the following:

* From the sbt prompt, run `set warForkOptions := ForkOptions()`
* Add to your project's *build.sbt*:
    ```scala
    warForkOptions := ForkOptions()
    ```

then run `warStart` again.

**Comment by earldouglas** [comment]
I tested this on Windows 10 with Java 17, and it looks like sbt-war is hosing the backslashes in Windows directory paths.

**Comment by earldouglas** [comment]
Reopening this until we get sbt-war.g8 updated too.

**Comment by earldouglas** [comment]
The fix is released in sbt-war version 5.0.0-M6.  sbt-war.g8 has been updated accordingly.

Please give it a try by either updating your project's *project/plugins.sbt* to point to sbt-war 5.0.0-M6, or by re-running `sbt new earldouglas/sbt-war.g8` to create a new project.

Note that to run the tests, you must first start the server:

```
$ sbt
> warStart
> test
```

**Comment by dylemma** [comment]
Confirmed fixed!
```
sbt:hello-sbt-war> test
[info] Passed: Total 1, Failed 0, Errors 0, Passed 1
[success] Total time: 6 s, completed Nov 18, 2024, 9:27:15 AM
```
![image](https://github.com/user-attachments/assets/176f0ec2-97de-4868-9c55-f4adab30f458)
